code_samples/data_migration/examples/create_segment.yaml

```
@@ -0,0 +1,24 @@
+-
+    type: segment_group
+    mode: create
+    name: 'Contractors'
+    identifier: contractors
+    references:
+        -
+            name: contractors_group_id
+            type: segment_group_id
+
+-
+    type: segment
+    mode: create
+    name: 'Painter'
+    identifier: painter
+    group:
+        id: 'reference:contractors_group_id'
```

While possible to use ID + reference, using `identifier` should be preferred.

```suggestion
        identifier: contractors
```